scripts/run_performance_job.py

```diff
@@ -251,10 +259,17 @@ def get_pre_commands(args: RunPerformanceJobArgs, v8_version: str):
         ]
 
     # ensure that the PYTHONPATH is set to the scripts directory
+    # TODO: Run scripts out of work item directory instead of payload directory
     if args.os_group == "windows":
-        helix_pre_commands += ["set PYTHONPATH=%HELIX_WORKITEM_ROOT%\\scripts%3B%HELIX_WORKITEM_ROOT%"]
+        helix_pre_commands += ["set PYTHONPATH=%HELIX_CORRELATION_PAYLOAD%\\scripts%3B%HELIX_CORRELATION_PAYLOAD%"]
```

Comment:
Do we need to escape this semicolon?

Reply:
Yes, because semicolon is used to split up all the precommands